sw/host/opentitanlib/src/bootstrap/mod.rs

```diff
@@ -111,6 +114,18 @@ impl<'a> Bootstrap<'a> {
         options: &BootstrapOptions,
         payload: &[u8],
     ) -> Result<()> {
+        if transport
+            .capabilities()?
+            .request(Capability::PROXY)
+            .ok()
+            .is_ok()
```

Comment:
This can just be `is_some()`, no?

Reply:
`request()` returns a custom struct, which happens to have an method called `ok()`, it is not related to `Option<>` and does not have a method called `is_some()`.

Thinking about it, maybe the `request()` method should be made to directly return a `Result<>`, rather than this intermediary custom structs, with which you can do little useful other than calling `ok()` to get the `Result<>`.